bullet/worlds/meshes/VLP16_base_1.dae

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
```

Comment:
We added those to make easier for someone checking this branch to reproduce some results / issues with the code. Do you think we can post these world elsewhere in particular? Otherwise I can just move them to my personal repo and add a link somewhere in a readme.